gpt4all-bindings/python/setup.py

```diff
@@ -59,13 +59,28 @@ def copy_prebuilt_C_lib(src_dir, dest_dir, dest_build_dir):
                     DEST_CLIB_DIRECTORY,
                     DEST_CLIB_BUILD_DIRECTORY)
 
+
+def get_long_description():
+    with open(
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), "README.md"),
+        encoding="utf8",
+    ) as fp:
```

Comment:
why not `from pathlib import Path` and then:
```suggestion
    with open(Path(__file__).parent / "README.md", encoding="utf8") as fp:
```

Reply:
Yeah, no reason not to I guess.